app/config/featureswitch/FeatureSwitching.scala

```diff
@@ -40,10 +40,13 @@ trait FeatureSwitching {
     }
   }
 
-  // TODO replace this function, or all uses of it, with the inverse of isEnabled
-  // It is currently used in NavBarFromNinoPredicate.scala, ForecastIncomeSummaryController.scala, ForecastTaxCalcSummaryController.scala
-  def isDisabled(featureSwitch: FeatureSwitchName): Boolean = {
-    sys.props.get(featureSwitch.name) orElse appConfig.config.getOptional[String](featureSwitch.name) contains FEATURE_SWITCH_OFF
+  // TODO: replace this function, or all uses of it, with the inverse of isEnabled
+  def isDisabled(featureSwitch: FeatureSwitchName, fs: List[FeatureSwitch]): Boolean = {
+    if (appConfig.readFeatureSwitchesFromMongo) {
+     !fs.exists(x => x.name.name == featureSwitch.name && x.isEnabled)
+    } else {
+      sys.props.get(featureSwitch.name) orElse appConfig.config.getOptional[String](featureSwitch.name) contains FEATURE_SWITCH_OFF
```

Comment:
Should be:
sys.props.get(featureSwitch.name) orElse appConfig.config.getOptional[String]("feature-switch.enable-"+ featureSwitch.name) contains FEATURE_SWITCH_OFF

because the names have a prefix in application.conf that isn't in FeatureSwitchName.name

Reply:
Need the same thing in the method: isEnabledFromConfig

Reply:
added required tests coverage and fix above;
remove not used FS keys as well